controllers/account/go.mod

```diff
@@ -5,41 +5,32 @@ go 1.20
 require (
 	github.com/go-logr/logr v1.2.3
 	github.com/labring/sealos v0.0.0
-	github.com/labring/sealos/controllers/infra v0.0.0
+	github.com/labring/sealos/controllers/common v0.0.0-20230329143406-e5a204154f23
```

Comment:
Manual change require to `github.com/labring/sealos/controllers/common v0.0.0`, do not lock to spec version (cannot lock in fact.)

Reply:
done
```
	github.com/labring/sealos v0.0.0
	github.com/labring/sealos/controllers/common v0.0.0
	github.com/labring/sealos/controllers/metering v0.0.0
	github.com/labring/sealos/controllers/user v0.0.0
replace (
	github.com/labring/sealos => ../../
	github.com/labring/sealos/controllers/common => ../../controllers/common
	github.com/labring/sealos/controllers/metering => ../../controllers/metering
	github.com/labring/sealos/controllers/user => ../../controllers/user
)
```